filebeat/docs/upgrading.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[upgrading-filebeat]]
-== Upgrading Filebeat
+== Upgrade Filebeat
```

Comment:
Why did you chose to include the product name in this heading, but in none of the others?

Reply:
Missed this during my search/replace, but will fix.